fractal_server/app/schemas/task.py

```diff
@@ -46,6 +49,9 @@ class TaskUpdate(_TaskBase):
     name: Optional[str]  # type:ignore
     input_type: Optional[str]  # type:ignore
     output_type: Optional[str]  # type:ignore
+    module: Optional[str]  # type:ignore
```

Comment:
idem

we should also decide what happens when a task is updated and if it even makes sense to update it manually.
I opened #192 for discussion on this issue